japanese/beanus/beanus.txt

```diff
@@ -0,0 +1,4 @@
+��̉��𗊂�ɁA�X�̒��ɂ��񂮂񕪂����ĂƁA�Y��Ȑ��������Ȃ��B
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.